src/main/java/org/datadog/jenkins/plugins/datadog/util/git/GitUtils.java

```diff
@@ -41,6 +44,7 @@ public final class GitUtils {
     public static final String GIT_REPOSITORY_URL = "GIT_URL";
     public static final String GIT_REPOSITORY_URL_ALT = "GIT_URL_1";
     public static final String GIT_BRANCH = "GIT_BRANCH";
+    public static final String GIT_BRANCH_ALT = "BRANCH_NAME";
```

Comment:
> [!CAUTION]
> In pull request builds, this can end up being `"PR-123"` (e.g. for a build on a branch that's part of pull request number 123). In those cases, the fallback env var `CHANGE_BRANCH` should be preferred over others. The precedence we generally use is `def branch = env.CHANGE_BRANCH ?: env.GIT_BRANCH ?: env.BRANCH_NAME` (Groovy example). Would you consider updating the logic as such? Thanks!